tooling/Dockerfile

```diff
@@ -1,16 +1,22 @@
-FROM fedora:30 AS kafkacat
+FROM fedora:34 AS kcat
 RUN dnf -y install gcc which gcc-c++ wget make git cmake
-ENV KAFKACAT_VERSION=1.5.0
+ENV KCAT_VERSION=1.5.0
 RUN dnf -y install cyrus-sasl-devel libcurl-devel libzstd-devel
-RUN git clone https://github.com/edenhill/kafkacat -b $KAFKACAT_VERSION --single-branch && \
-    cd kafkacat && \
+RUN git clone https://github.com/edenhill/kcat -b $KCAT_VERSION --single-branch && \
+    cd kcat && \
     ./bootstrap.sh
 
-FROM fedora:30
+FROM fedora:34
 RUN dnf -y install jq httpie mycli python3-pip gcc redhat-rpm-config python-devel libpq-devel &&\
     pip install pgcli &&\
     dnf clean all
-COPY --from=kafkacat /kafkacat/kafkacat /usr/bin/kafkacat
+COPY --from=kcat /kcat /usr/bin/kcat
```

Comment:
@ani-sha Could you please add a symlink so `kafkacat` command would still work?

Reply:
Good idea!